buildenv/jenkins/docker-slaves/x86/centOS/Dockerfile

```diff
@@ -0,0 +1,219 @@
+  && cd .. \
+  && rm -rf ant-contrib
+
+RUN cd /usr/src/ \
+  && wget http://developer.download.nvidia.com/compute/cuda/7.5/Prod/local_installers/cuda_7.5.18_linux.run \
+  && chmod +x cuda_7.5.18_linux.run \
+  && /usr/src/cuda_7.5.18_linux.run --silent --toolkit --override \
+  && rm /usr/src/cuda_7.5.18_linux.run
+
+# add user home/jenkins
+RUN useradd -ms /bin/bash ${USER} \
+  && mkdir /home/${USER}/.ssh/
+
+COPY authorized_keys /home/${USER}/.ssh/authorized_keys
+
+RUN chown -R ${USER}:${USER} /home/${USER} \
+  && chmod 644 /home/${USER}/.ssh/authorized_keys
+
+# set up sshd config
+RUN mkdir /var/run/sshd \
+  && sed -i 's/#PermitRootLogin/PermitRootLogin/' /etc/ssh/sshd_config \
+  && sed -i 's/#RSAAuthentication.*/RSAAuthentication yes/' /etc/ssh/sshd_config \
+  && sed -i 's/#PubkeyAuthentication.*/PubkeyAuthentication yes/' /etc/ssh/sshd_config
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# Download and install boot JDK from AdoptOpenJDKs
+
+RUN ln -s java-1.7.0 /usr/lib/jvm/java-7-openjdk-amd64
+
+RUN mkdir /usr/lib/jvm/adoptojdk-java-80 \
+  && cd /usr/lib/jvm/adoptojdk-java-80 \
+  && wget -O bootjdk8.tar.gz https://github.com/AdoptOpenJDK/openjdk8-openj9-releases/releases/download/jdk8u181-b13_openj9-0.9.0/OpenJDK8-OPENJ9_x64_Linux_jdk8u181-b13_openj9-0.9.0.tar.gz \
+  && tar -xzf bootjdk8.tar.gz \
+  && rm -f bootjdk8.tar.gz \
+  && ls | grep -i jdk | xargs -I % sh -c 'mv % bootjdk8' \
+  && mv bootjdk8/* /usr/lib/jvm/adoptojdk-java-80
+
+RUN mkdir /usr/lib/jvm/adoptojdk-java-90 \
+  && cd /usr/lib/jvm/adoptojdk-java-90 \
+  && wget -O bootjdk9.tar.gz https://github.com/AdoptOpenJDK/openjdk9-openj9-releases/releases/download/jdk-9.0.4%2B12_openj9-0.9.0/OpenJDK9-OPENJ9_x64_Linux_jdk-9.0.4.12_openj9-0.9.0.tar.gz \
+  && tar -xzf bootjdk9.tar.gz \
+  && rm -f bootjdk9.tar.gz \
+  && ls | grep -i jdk | xargs -I % sh -c 'mv % bootjdk9' \
+  && mv bootjdk9/* /usr/lib/jvm/adoptojdk-java-90
+  
+RUN mkdir /usr/lib/jvm/adoptojdk-java-10 \
+  && cd /usr/lib/jvm/adoptojdk-java-10 \
+  && wget -O bootjdk10.tar.gz https://github.com/AdoptOpenJDK/openjdk10-openj9-releases/releases/download/jdk-10.0.2%2B13_openj9-0.9.0/OpenJDK10-OPENJ9_x64_Linux_jdk-10.0.2.13_openj9-0.9.0.tar.gz \
+  && tar -xzf bootjdk10.tar.gz \
+  && rm -f bootjdk10.tar.gz \
+  && ls | grep -i jdk | xargs -I % sh -c 'mv % bootjdk10' \
+  && mv bootjdk10/* /usr/lib/jvm/adoptojdk-java-10
+
+RUN cd /home/${USER} \
+  && wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz \
+  && tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2 \
+  && rm -f freemarker.tgz
+
+RUN cd /usr/src/ \
+  && wget https://fossies.org/linux/misc/autoconf-2.69.tar.gz \
+  && tar -xzf autoconf-2.69.tar.gz \
+  && rm -f autoconf-2.69.tar.gz \
+  && yum remove autoconf -y \
```

Comment:
if line 33 `autoconf \` is removed, then `yum remove autoconf -y` can also be removed.